package.json

```diff
@@ -31,7 +31,7 @@
     "typescript": "~4.0.5"
   },
   "scripts": {
-    "prepare": "lerna run prepare",
+    "prepare": "lerna --concurrency=2 run prepare",
```

Comment:
maybe move out to a separate PR/issue ?
or have a way to provide parameters/override it